### DISCUSSION TITLE
a few initial questions

### DISCUSSION THREAD

**Discussion by omega3** [KoljaB/RealtimeTTS#35] [General]
I have a few initial questions to see where are you heading. I used couqui-tts to produce audio from articles for language learning purposes. So I wanted fast, low-resources English models with correct English pronunciation, without all artificially added emotions, for producing TTS off-line, on low quality hardware (Intel CPU with integrated graphic).

1. Is it suppose to replace coqui-tts? 
2. Is it going to be possible to use it in command line to produce TTS with models from coqui-tts (for example model 21: tts_models/en/vctk/vits)?
3. Are you going to support at least some ssml tags or any other way to insert breaks / pauses and speed control?

**Comment by KoljaB** [comment]
1. Currently realtime support for coqui **XTTS model** only. There are lots of cases where you would want to use coqui tts but not RealtimeTTS (for example using VITS model). So no, it's not supposed to replace coqui-tts, there are use cases against it.
2. Currently no CLI. Coqui already has one and this lib focuses on realtime. Coqui models must be loaded which causes significant delay, so a CLI would not be possible to do in realtime without some complex client server infrastructure.
3. Probably not that soon. From the currently supported engines only Azure supports ssml. Speed is supported for Azure+Coqui engines, the others don't support it. Breaks and pauses can sometimes be done with special chars depending on the engine like "..." or "-"

**Comment by omega3** [comment]
I installed via pip to use coqui voices. How to run this window where you paste text?

**Comment by KoljaB** [comment]
This is the [pyqt6_speed_test.py](https://github.com/KoljaB/RealtimeTTS/blob/master/tests/pyqt6_speed_test.py) file in the tests directory
